doc/deployment-schema.json

```diff
@@ -0,0 +1,310 @@
+                                            ],
+                                            "properties": {
+                                                "version": {
+                                                    "type": "string",
+                                                    "examples": [
+                                                        "1.0"
+                                                    ]
+                                                },
+                                                "type": {
+                                                    "$ref": "#/definitions/moduleType"
+                                                },
+                                                "status": {
+                                                    "$ref": "#/definitions/status"
+                                                },
+                                                "restartPolicy": {
+                                                    "$ref": "#/definitions/restartPolicy"
+                                                },
+                                                "env": {
+                                                    "$ref": "#/definitions/env"
+                                                },
+                                                "settings": {
+                                                    "$ref": "#/definitions/moduleSettings"
+                                                }
+                                            }
+                                        }
+                                    },
+                                    "additionalProperties": false
+                                }
+                            }
+                        }
+                    }
+                },
+                "$edgeHub": {
+                    "type": "object",
+                    "title": "Configuration for the edgeHub module",
+                    "required": [
+                        "properties.desired"
+                    ],
+                    "properties": {
+                        "properties.desired": {
+                            "type": "object",
+                            "required": [
+                                "schemaVersion",
+                                "routes"
+                            ],
+                            "properties": {
+                                "schemaVersion": {
+                                    "type": "string",
+                                    "examples": [
+                                        "1.0"
+                                    ]
+                                },
+                                "routes": {
+                                    "type": "object",
+                                    "patternProperties": {
+                                        "^.+$": {
+                                            "type": "string",
```

Comment:
Do we have a regular expression to define the routes instead of just pure string here?

Reply:
I added a pattern